pull_request.yml

```diff
@@ -0,0 +1,13 @@
+name: "Pull Request Docs Check"
+
+on: 
+- pull_request
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ammaraskar/sphinx-action@master
```

Comment:
I haven’t looked more but when you showed me this I thought it was a build script. Is there a syntax check we could run for PRs, and then run the build action on merge?